src/components/app/details/cIDetails/types.tsx

```diff
@@ -10,14 +10,15 @@ export interface CIPipeline {
 export interface BuildDetails {
     triggerHistory: Map<number, History>
     fullScreenView: boolean
-    synchroniseState: (triggerId: number, triggerDetails: History) => void
+    synchroniseState: (triggerId: number, triggerDetails: History, triggerDetailsError: any) => void
```

Comment:
if possible use specific type instead of any

Reply:
There is a interface called ServerErrors but the response from api that i was checking also had another field called status can I extend it by keeping status optional in ServerError?